docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -0,0 +1,311 @@
+
+1. **Create an entry point script**: Create a new file in `<project_root>/src/iris_databricks` named `databricks_run.py`. Copy the following code to this file:
+
+```python
+import argparse
+import logging
+
+from kedro.framework.project import configure_project
+from kedro.framework.session import KedroSession
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--env", dest="env", type=str)
+    parser.add_argument("--conf-source", dest="conf_source", type=str)
+    parser.add_argument("--package-name", dest="package_name", type=str)
+
+    args = parser.parse_args()
+    env = args.env
+    conf_source = args.conf_source
+    package_name = args.package_name
+
+    # https://kb.databricks.com/notebooks/cmd-c-on-object-id-p0.html
+    logging.getLogger("py4j.java_gateway").setLevel(logging.ERROR)
+    logging.getLogger("py4j.py4j.clientserver").setLevel(logging.ERROR)
+
+    configure_project(package_name)
+    with KedroSession.create(env=env, conf_source=conf_source) as session:
+        session.run()
+
+
+if __name__ == "__main__":
+    main()
+```
+
+2. **Define a new entry point**: Open `<project_root>/src/setup.py` in a text editor or IDE and add a new line in the definition of the `entry_point` tuple, so that it becomes:
+
+```python
+entry_point = (..., "databricks_run = <package_name>.databricks_run:main")
+```
+
+Remember to replace <package_name> with the correct package name for your project.
+
+This process adds an entry point to your project which can be used to run it on Databricks.
+
+```{note}
+Because you are no longer using the default entry-point for Kedro, you will not be able to run your project with the options it usually provides. Instead, the `databricks_run` entry point in the above code and in the Databricks Iris starter contains a simple implementation of two options:
+- `--package_name` (required): the package name (defined in `setup.py`) of your packaged project.
+- `--env`: specifies a [Kedro configuration environment](../../configuration/configuration_basics.md#configuration-environments) to load for your run.
+- `--conf-source`: specifies the location of the `conf/` directory to use with your Kedro project.
+```
+
+### Plan your project's output
+
+You will not be able to directly see your project's log output when it is run as a Databricks job. You will need to decide on and implement a strategy for storing your project's results. Some examples are:
+
+- Write the result of a data engineering pipeline to a managed delta table.
+- Upload metrics from a machine learning model to MLflow.
+- Save a dataset as a CSV file in DBFS.
+
+In this guide, your project will save its logs to a location in DBFS (`/dbfs/FileStore/iris-databricks/info.log`).
+
+**The `databricks-iris` starter is already configured to save logs in this location.** The configuration used to achieve this can be found in the `conf/base/logging.yml` file.
```

Comment:
On the other hand, related to my comment above, I like this approach of not explaining what's in `logging.yml` to avoid interrupting the how-to. But maybe we could add a note at the bottom.